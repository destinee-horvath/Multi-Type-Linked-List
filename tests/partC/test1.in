NEW 4
1
2
3
4
NEW 3
Numbers
{0}
end
VIEW 0
TYPE 0
VIEW 1
TYPE 1
VIEW ALL
VIEW-NESTED 0
VIEW-NESTED 1
