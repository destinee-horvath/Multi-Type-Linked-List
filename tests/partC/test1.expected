List 0: 1 -> 2 -> 3 -> 4
Nested 1: Numbers -> {List 0} -> end
1 -> 2 -> 3 -> 4
int -> int -> int -> int
Numbers -> {List 0} -> end
string -> reference -> string
Number of lists: 2
List 0
Nested 1
1 -> 2 -> 3 -> 4
Numbers -> {1 -> 2 -> 3 -> 4} -> end
